keras_rcnn/layers/object_detection/object_proposal.py

```diff
@@ -0,0 +1,59 @@
+import keras.backend
```

Comment:
I was wondering what the reason was for the underscore in the name, none of the other modules have this naming convention?